src/NServiceBus.Core.Tests/ConventionsTests.cs

```diff
@@ -1,43 +1,11 @@
 ﻿namespace NServiceBus.Core.Tests
 {
-    using System;
     using NUnit.Framework;
 
     [TestFixture]
     public class ConventionsTests
     {
-        [Test]
-        public void Should_use_TimeToBeReceived_from_bottom_of_tree()
```

Comment:
By looking from top down over the changes I couldn't figure out why these tests are no longer needed. Can you explain?